api/README.md

```diff
@@ -0,0 +1,8 @@
+### Notice
+
+Do not change .pb.go file directly. We need to change the corresponding .proto file and run the following command to generate.
+```
+$ make generate
+```
+
+More infomation about [protobuf](https://github.com/google/protobuf)
```

Comment:
It's not a big issue, but this link styling looks strange to me. I would write this as
More infomation about protobuf: https://github.com/google/protobuf
OR
Click [here](https://github.com/google/protobuf) for more information about protobuf.